<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowControleur</class>
 <widget class="QMainWindow" name="MainWindowControleur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controleur</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Date :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Visite :</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="3">
     <widget class="QPushButton" name="pushButtonModifier">
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QTimeEdit" name="timeEditFin"/>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Modification :</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QTextEdit" name="textEditCommentaire"/>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Heure debut :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Commentaire :</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QDateEdit" name="dateEdit"/>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QListWidget" name="listWidgetVisite"/>
    </item>
    <item row="7" column="1">
     <widget class="QTimeEdit" name="timeEditDebut"/>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Heure fin :</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Etat :</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1" colspan="2">
     <widget class="QComboBox" name="comboBoxEtat"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>319</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
